CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ These are the section headers that we use:
 
 - Added `max_records` argument to `pull()` method for `RemoteFeedbackDataset`.([#4074](https://github.com/argilla-io/argilla/pull/4074))
 - Added functionality to push your models to huggingface hub with `ArgillaTrainer.push_to_huggingface` ([#3976](https://github.com/argilla-io/argilla/pull/3976)).
+- Added `POST /api/v1/datasets/:dataset_id/records/search` endpoint to search for records without user context, including responses by all users. ([#4143](https://github.com/argilla-io/argilla/pull/4143))
```

Comment:
I think we should also add changes to the limit for existing endpoints.

Reply:
Yes, let me add it.

Reply:
Done!